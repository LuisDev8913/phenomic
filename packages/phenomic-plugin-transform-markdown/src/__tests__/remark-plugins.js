import renderer from "react-test-renderer"

import remarkPlugins from "../remark-plugins.js"

it("should render markdown as react component", () => {
  const md = remarkPlugins(
    "## Test\n[link](href)\n```js\nconsole.log(window)\n```",
  ).contents
  const tree = renderer.create(md).toJSON()
  expect(tree).toMatchSnapshot()
})
