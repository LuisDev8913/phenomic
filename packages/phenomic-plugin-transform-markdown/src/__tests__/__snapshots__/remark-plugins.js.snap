// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render markdown as react component 1`] = `
<div>
  <h2
    id="test"
  >
    <a
      href="#test"
    />
    Test
  </h2>
  
  
  <p>
    <a
      href="href"
    >
      link
    </a>
  </p>
  
  
  <pre>
    <code
      className="hljs js"
      style={
        Object {
          "display": "inline",
        }
      }
    >
      <span
        className={
          Array [
            "hljs-built_in",
          ]
        }
      >
        console
      </span>
      .log(
      <span
        className={
          Array [
            "hljs-built_in",
          ]
        }
      >
        window
      </span>
      )
      
    </code>
  </pre>
</div>
`;
